openhcl/virt_mshv_vtl/src/processor/mshv/x64.rs

```diff
@@ -407,12 +316,12 @@ impl BackingPrivate for HypervisorBackedX86 {
             .set_sints(this.backing.next_deliverability_notifications.sints() | sints);
     }
 
-    fn hv(&self, vtl: GuestVtl) -> Option<&ProcessorVtlHv> {
-        self.hv.as_ref().map(|arr| &arr[vtl])
+    fn hv(&self, _vtl: GuestVtl) -> Option<&ProcessorVtlHv> {
```

Comment:
These can probably be made to go away with some more refactoring.

Reply:
Ehh, maybe not, I dunno

Reply:
Yeah, it's on my follow up list to try.